src/utils/config/fantom.ts

```diff
@@ -6,28 +6,27 @@ import {
 } from '../../types';
 
```

Comment:
@SmoothBot this file contains a bunch of network constants. I've updated a bunch of them for RoboVault but left some alone. Could please have a look at the file (what i have change, and what i haven't) and see if anything looks wrong?

Reply:
ok cool. I'm not sure what it does but i'll dig in and figure out what its for